### ISSUE TITLE
[review]: learn-near-accounts-by-creating-a-named-testnet-account

### ISSUE THREAD

**Issue by ShaunSHamilton** [freeCodeCamp/near-curriculum#45]
# Overview

## Lessons

1\. Typo
```diff
- ...you need, run...
+ ...you need. Run...
```
4\. Need to add `near-cli` to `.gitpod.Dockerfile`
7\. Typo
```diff
- ...locally that allow...
+ ...locally which allow...
```
10\. Do you want me to create an `<account>.json` file (description)? Or, a an `<account>.testnet.json` file?
11\. Unfinished last sentence?
26\. Test permission error:
<img width="523" alt="image" src="https://user-images.githubusercontent.com/51722130/233439709-e07d51c4-5e94-42ac-bd01-b50accb3e770.png">
33\. First sentence seems very unrelated.
39\. Test permission error
<img width="538" alt="image" src="https://user-images.githubusercontent.com/51722130/233442220-ef002415-090d-44c7-a7e2-49bb03a1540e.png">

**placeholder**: _reached lesson 70_

**Comment by moT01** [comment]
Thanks for taking a look @ShaunSHamilton 👍 

Did you run it in gitpod? and what does `placeholder: reached lesson 70` mean? Did you just stop, or did the course get stuck or something?

**Comment by ShaunSHamilton** [comment]
Yes, I ran this in Gitpod. Yes, I had to stop for the day, so I will need to finish the review later.
